docs/operator/updating.md

```diff
@@ -0,0 +1,38 @@
+# Updating
+
+## Updating the Operator
+
+Updating the operator is as simple as re-running:
+
+```sh
+kubectl apply --server-side -k github.com/authzed/spicedb-operator/config
+```
+
+whenever this is an update available.
+
+## Updating Managed SpiceDBClusters
+
+### Automatic Updates
+
+Every release of SpiceDB Operator comes with a version of SpiceDB that it considers the "default" version.
+This default version is used for every `SpiceDBCluster` that does not specify a specific `image` in its `.spec.config`.
+When either the operator or its config file is updated, every cluster under management will be rolled to the new default version.
+
+This is recommended primarily for development environments.
+
+If you wish to have truly zero-touch upgrades, you can automate the updating of the operator with standard git-ops tooling like [Flux] or [ArgoCD].
```

Comment:
Or use dedicated (link to it)

Reply:
Dedicated has both options though (auto updates and manually deciding when to upgrade), it's not really an alternative just for auto updates.

Instead I put a note at the top of this section.